ngraph_creator/src/NgraphNetworkCreator.cpp

```diff
@@ -104,7 +104,7 @@ void NgraphNetworkCreator::getSupportedOperations(std::vector<bool>& supportedOp
 
 bool NgraphNetworkCreator::validateOperations() {
     for (int i = 0; i < mModelInfo->getOperationsSize(); i++) {
-        if (!mOperationNodes[i] || !mOperationNodes[i]->validate()) {
+        if (!mOperationNodes[i] || !mOperationNodes[i]->validateForPlugin()) {
```

Comment:
@akodanka Why is validation restricted only to inputs?

Reply:
The reason is that each operation is dependent on the input operands to the node, and the output of the node goes as input to the next operation(or result), where it gets validated. This assumption has been holding good so long.
I think this is a nice observation though, and we can discuss and plan on validating the outputs too, if required.
But again, that would be outside the scope of this PR.